pages/roadmap/[...slug].tsx

```diff
@@ -139,34 +141,35 @@ export default function RoadmapPage(props: InferGetServerSidePropsType<typeof ge
       setIsPendingChildrenLoading(false);
     };
     fetchPendingChildren();
-  }, [issue_number, repo, owner, isRootIssueLoading, pendingChildrenState.value]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [issue_number, repo, owner, isRootIssueLoading, pendingChildrenState.length]);
 
   /**
    * Add asyncIssueData items to issueDataState
    */
   useEffect(() => {
     const issueData = issueDataState.get({ noproxy: true }) as IssueData;
     const asyncIssues = asyncIssueDataState.get();
-    if (asyncIssues.length === 0) {
+    const newIssueData = asyncIssueDataState[0];
+    if (asyncIssues.length === 0 || newIssueData == null) {
+      asyncIssueDataState[0].set(none);
```

Comment:
handling the same as pendingChildrenState, though I didn't notice an issue here. Either way, if it's empty, this won't hurt, if it's not empty, `[0]` is invalid and needs deleted.